src/Card/CardDeck.java

```diff
@@ -45,55 +71,55 @@ public void WriteOutEverything(){
     
     // put cards into easy cards
     // if there is no enough card the user has to add more cards 
-    public void loadEasyCards(){
-        if(cards.size() >= easyCardNumber){
+    public void loadHarderCards(){
+        if(cards.size() >= harderCardNumber){
             // if the size is the same with the number put all the cards into the easycards
-            if(cards.size() == easyCardNumber){
-                easyCards.putAll(cards);
-                WriteOutEasyCards();
+            if(cards.size() == harderCardNumber){
+                harderCards.putAll(cards);
+                WriteHarderCards();
             }else{
-                // if
+                // String array is just help to choose cards
                 String[] words = new String[cards.size()];
                 int i = 0;
                 for(String key : cards.keySet()){
                     words[i] = key;
                     ++i;
                 }
-                String[] readyWords = new String[easyCardNumber];
+                String[] readyWords = new String[harderCardNumber];
                 readyWords = ChooseRandomWords(words);
-                // now i have the different word and I put them into the easy
-                // word[i] == dog
+                // now i have the different words int readyWords and I put them into the easyCards
                 for(int k = 0; k < readyWords.length; ++k){
                     String english = readyWords[k];
                     String hungarian = cards.get(readyWords[k]);
-                    easyCards.put(english, hungarian);
+                    harderCards.put(english, hungarian);
                 }
                 // Delete this function!
-                WriteOutEasyCards();
+                WriteHarderCards();
             }
+        // if there is not enough card, tell user to upload more cards
         }else{
-            System.out.println("Sorry, you do not have enough card, please put at least " + easyCardNumber + " cards");
+            System.out.println("Sorry, you do not have enough card, please put at least " + harderCardNumber + " cards");
```

Comment:
The user will not see console of Java when they run our program. So this should be changed in order to be visible on a  certain panel, so that the user will notice it as well.